Readme.md

```diff
@@ -35,21 +35,34 @@ We recommend to use python-virtualevn (virtualenv and virtualenv-wrapper)
 
 ## Features
 
- * tc_aws.loaders.s3_loader
- * tc_aws.loaders.presigning_loader
- * tc_aws.result_storages.s3_storage
- * tc_aws.storages.s3_storage
+ * *tc_aws.loaders.s3_loader* - takes a S3 key path and optional bucket name, and downloads the file through the S3 API.
+ * *tc_aws.loaders.presigning_loader* - instead of downloading via the API, generates a signed link to the file on S3, then feeds it to the Thumbor's regular http loader. This will likely be more performant, as it avoids async issues with the boto library (see [#22](https://github.com/thumbor-community/aws/pull/22) and [#14](https://github.com/thumbor-community/aws/issues/14).
+ * *tc_aws.result_storages.s3_storage*
+ * *tc_aws.storages.s3_storage*
+ 
+Why use S3-specific loaders rather than just giving the S3 HTTP url to Thumbor? If your S3 files are private and you sign the urls, the query strings will vary and Thumbor will not be able to cache the image.
 
 Additional Configuration values used:
 
 ```.ini
-TC_AWS_REGION='eu-west-1' # AWS Region
+# AWS Region the bucket is located in. 
+TC_AWS_REGION='eu-west-1' 
+# A custom AWS endpoint.
+TC_AWS_ENDPOINT=''
+# You might need to set this to True for some regions, such as eu-central-1.
+S3_USE_SIGV4=False
```

Comment:
Yes, it's in boto. I frankly have had enough with various auth issues that I've had with AWS over the years, and I'm not sure why boto still doesn't take care of that for me. Still, for years, the only way to use the Frankfurt S3 region in my experience was to set this flag. See https://github.com/APSL/docker-thumbor/issues/13. Up to you if this should be mentioned here.